tests/test-content/js/layout/placeholder.js

```diff
@@ -0,0 +1,36 @@
+// Copyright 2018 Endless Mobile, Inc.
+
+/* exported Placeholder */
+
+const GObject = imports.gi.GObject;
+const Gtk = imports.gi.Gtk;
+
+const Config = imports.app.config;
+const Module = imports.app.interfaces.module;
+
+/**
+ * Class: Placeholder
+ * This is a modified version of the SDK Placeholder version
+ */
+var Placeholder = new Module.Class({
+    Name: 'Layout.Placeholder',
+    Extends: Gtk.Frame,
+
+    Properties: {
+        'text': GObject.ParamSpec.string('text', 'Text', 'Placeholder text',
+            GObject.ParamFlags.READWRITE | GObject.ParamFlags.CONSTRUCT_ONLY,
+            'Overridden'),
+    },
+
+    _init: function (props={}) {
```

Comment:
Use the new method syntax: `_init(props={}) {`

Reply:
Good catch.